recipe/meta.yaml

```diff
@@ -1,54 +1,75 @@
 {% set name = "sktime" %}
 {% set version = "0.12.1" %}
 
-
 package:
-  name: {{ name|lower }}
+  name: sktime-suite
   version: {{ version }}
 
 source:
   url: https://github.com/alan-turing-institute/{{ name }}/archive/v.{{ version }}.tar.gz
   sha256: 0e39ce6f7bbe59e3013a3a5f756aa6234f93a570ad6a1cfacebf78872b461529
 
 build:
-  number: 1
-  skip: true  # [(py==37 and aarch64) or py>39]
+  number: 2
+  skip: true  # [(py==37 and aarch64)]
+  skip: true  # [py<37 or py>39]
+
+# This section is required to correctly rerender skip statement
+requirements:
+  host:
+    - python
+  run:
+    - python
 
 outputs:
   - name: sktime
-    script: build_script.sh
+    script: build_script.sh   # [unix]
+    script: build_script.bat  # [win]
     requirements:
       build:
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - llvm-openmp  # [osx]
       host:
         - python
         - pip
         - setuptools
+        - wheel
+        - toml
+        - build
       run:
         - python
         - deprecated >=1.2.13
-        - {{ pin_compatible('numpy') }}
+        - numpy >=1.21.0,<1.22
         - pandas >=1.1.0,<1.5
         - numba >=0.53
         - scikit-learn >=0.24.0,<1.2.0
         - statsmodels >=0.12.1
         - scipy <1.9.0
       run_constrained:
+        - dtw-python
+        - esig ==0.9.7
+        - filterpy >=1.4.5
+        - hcrystalball >=0.1.9
         - matplotlib-base >=3.3.2
         - pmdarima >=1.8.0,!=1.8.1
+        - prophet >=1.0,<1.1
+        - pykalman >=0.9.5
+        - pyod >=0.8.0
+        - pystan ==2.19.1.1
         - scikit-posthocs >=0.6.5
         - seaborn >=0.11.0
-        - tsfresh >=0.17.0
+        - statsforecast>=0.5.2
         - stumpy >=1.5.1
         - tbats >=1.1.0
-        - prophet >=1.1  # [not aarch64]
-        - filterpy >=1.4.5
-        - pykalman >=0.9.5
+        - tensorflow
+        - tensorflow-probability
```

Comment:
I think any optional dependencies without constraints shouldn't be in `run_constrained` but should instead just be in the `sktime-all-extras` package which is actually installable unlike `run_constrained` specs.

Reply:
@carterbox any reason to remove tsfresh?

@dhirschfeld are you suggesting to drop `run_constrained` section for output=sktime or just to drop `tensorflow` and `tensorflow-probability` from the list? I am not entirely sure what should go into `run_constrained` to be honest and [the docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained) aren't very helpful in this case:

> Packages that are optional at runtime but must obey the supplied additional constraint if they are installed.

Reply:
> @carterbox any reason to remove tsfresh?

I didn't; `tsfresh` is still listed.

> I think any optional dependencies without constraints shouldn't be in run_constrained but should instead just be in the sktime-all-extras package which is actually installable unlike run_constrained specs.

The reason that I included unconstrained package names in the run_constrained section is for readability. Packages with no constraint doesn't affect the solver, but they makes it much easier to compare this list with [the upstream list](https://github.com/alan-turing-institute/sktime/blob/67f81b109bc46d5a8bb14d4e476896397bfd57ef/pyproject.toml#L54). It also allows you to see which packages are missing from the `all-extras` metapackage without loading the upstream page.

> I am not entirely sure what should go into run_constrained to be honest and [the docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained) aren't very helpful in this case:

`requirements/run` says "these packages must be installed when sktime is installed". `requirements/run_constrained` says "IF these packages are installed when sktime is installed, then they must match the provided version specification".